src/etc/tidy.py

```diff
@@ -24,6 +24,7 @@
 
 interesting_files = ['.rs', '.py', '.js', '.sh', '.c', '.h']
 uninteresting_files = ['miniz.c', 'jquery', 'rust_android_dummy']
+stable_whitelist = []
```

Comment:
It surprises me that the whitelist is empty. std and core are stable and AFAICT nothing else encodes the knowledge that they're allowed to be stable, so surely they should be in the whitelist?